### ISSUE TITLE
UX parity between prototypes & coded client

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop-client#261]
_Related Issues from the UX Repo:_
• **[Client Content Design/Inventory](https://github.com/freedomofpress/securedrop-ux/issues/17)**
     » Table in first comment has full inventory list of icons w/ SVG art for completed ones.
     » Cut-art for elsewhere in the Client chrome will be included in related task Issues
• **[Client Visual Design](https://github.com/freedomofpress/securedrop-ux/issues/31)**
• **[Client Interaction Design](https://github.com/freedomofpress/securedrop-ux/issues/18)**

---
**»» [Zeplin Project for this Issue](https://app.zeplin.io/project/5c807ea562f734bd2756b243/dashboard) ««**

- [x] Re-Factor **Client Grid** (#259) 
- [ ] Apply **Source Pane Styles**
   • _Within scope of features already implemented_
   • Gradient details for background that spans this and the Messaging Pane, in its Styles Issue
   • No unique "Issue" for styling most of this, just follow Zeplin
- [ ] Apply **Conversation Pane Styles** (#272)
   • _Within scope of features already implemented_
- [ ] Apply **Reply Pane Styles** (#268)
   • Replace boring button w/ awesome airplane
- [x] Apply **Auth Widget Styles**/Interaction(s) (#264)
   • "interactions" within the scope of existing functionality = dropdown to signoff, vs static button
   • Placeholder initials and username instead of FName/LName, until Admin interface expanded to support FN/LN
- [ ] Apply **Refresh Widget Styles**/Interactions
   • Text styles for Refresh timestamp, in Zeplin mox
   • Static & Active icons, in Content Inventory (link above)
- [ ]  Configure Qubes SVS VM window for Workstation (#260) 
• Make the window match the spec'd purple, spec'd stroke color, spec'd dropshadow
• Spec icons style
• Spec typeface for VM Name
• Update VM naming convention for SVS to more user-friendly [SD Workstation]
• Not in scope for current sprint... 😢

**Comment by ninavizz** [comment]
^ Updated above punch-list to reflect ID Badge implementation blockage flagged in [SD-4251](https://github.com/freedomofpress/securedrop/issues/4251), and recommended workaround until the SD-4251 resolved.

Acceptance of this Epic is not dependent on the ID Badge thingy being unblocked. Just FYI! @creviera

**Comment by ninavizz** [comment]
For "Reply Pane Styles," SVG code for Awesome Airplane™ button:

`<?xml version="1.0" encoding="UTF-8"?>
<svg width="56px" height="39px" viewBox="0 0 56 39" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    
    <title>AeroPlane</title>
    <desc>Created with Sketch.</desc>
    <g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="VD_ComposeBox" transform="translate(-707.000000, -109.000000)" fill="#9814C8">
            <polygon id="AeroPlane" transform="translate(736.473347, 129.347086) rotate(-8.000000) translate(-736.473347, -129.347086) " points="708.205702 120.847093 719.410273 130.430186 743.788351 122.783606 723.799509 134.318549 720.095201 145.921376 729.046224 138.120065 737.160532 145.692965 764.740991 112.772796"></polygon>
        </g>
    </g>
</svg>`

**Comment by ninavizz** [comment]
^ @creviera BUMP! Added style tidbits on Refresh button, inline with text on OG comment. Created new issue for Auth Widget, which I'll be completing momentarily.

**Comment by ninavizz** [comment]
@creviera do you and @redshiftzero think there's enough from me between this Issue and the details provided in all the linked issues, for documentation/reference on this sprint's needs?

**Comment by eloquence** [comment]
Thanks for filing these, @ninavizz! I've added them to the [board](https://github.com/orgs/freedomofpress/projects/1), replacing the placeholder cards, and to the [spreadsheet](https://docs.google.com/spreadsheets/d/1RNPKas1OuaqnIOs8jVUjyAokq3TozXy-lahIqpxiE6U/edit#gid=0). Per the current plan we still need to flesh out issues for :

- Apply Source Pane Styles
- Apply Messaging Pane Styles

Jen/Nina/Allie, please jump in if you think that breakdown no longer makes sense. :)

**Comment by ninavizz** [comment]
^ @creviera I refrained from creating unique Issues for the _Source Pane Styles_ and the _Messaging Pane Styles_, figuring the screens in Zeplin should provide adequate guidance for a first pass. What do you think? Happy to create unique issues for those, if more info would help. 😃

**Comment by ninavizz** [comment]
Hey folks—I spaced on including the Login screen, in my list above—but recall discussing it with Erik and Allie, last Thursday. I also don't see an Issue for it—was one created that I suck at finding, @eloquence? Asking, cuz I wanted to know if for Beta we're still planning to do the combobox/dropdown for known usernames for a specific Client instance: https://invis.io/J3R7XDP2DSZ#/354546900_SignIn-Online

**Comment by eloquence** [comment]
Login UI changes are out of scope for the current sprint. For the next sprint, we may want to tackle the actionable changes identified in https://github.com/freedomofpress/securedrop-ux/issues/48#issuecomment-475455219 as a narrowly scoped issue.

We've not decided whether we want to include the "list of users" for offline mode. I suspect that we may reach a point where it's easier to have that list than to suppress all state in offline mode. For this reason I would love to see a final design with the dropdown, but we'll likely not implement it immediately.

**Comment by ninavizz** [comment]
Dev-actionable things for the near-term need to be resolved in a solution that scales-out to what's desired for Beta... so what we identified as "actionable" for the UX Kid to work on in this sprint, she still needs to consider as a minimization of what is the preferred solution for Beta. Hence, asking the prior question. 😸 

I'm personally in favor of preserving the drop-down for offline mode, as I suspect it'd feel weird for users if it went away.

**Comment by ninavizz** [comment]
Ohai this issue is sooooo obsolete; much has changed, so, closing!
